src/map/map.ts

```diff
@@ -198,12 +201,15 @@ interface MarkerData {
  * plot markers.
  */
 export class PropertiesMap {
-    /** Callback fired when the plot is clicked and a new point is selected */
+    /** Callback fired when the plot is clicked and the position of the active marker changes */
     public onselect: (indexes: Indexes) => void;
-    /** Callback fired when a guid is removed from the map */
-    public onremove: (removedGUID: string) => void;
```

Comment:
Why have you removed the `onremove` call? Now under 3D the deleted widgets still remain as markers on the 3D map, so I'm assuming the removal of widgets is no longer effectively communica
<img width="1280" alt="Screen Shot 2020-07-06 at 11 23 36 AM" src="https://user-images.githubusercontent.com/47536110/86577967-57043e80-bf7b-11ea-85ca-73ade077da9d.png">
ted.

Reply:
Also, after the marker is not removed, the activating of markers on the right does not correspond with the markers 
<img width="1280" alt="Screen Shot 2020-07-06 at 11 25 27 AM" src="https://user-images.githubusercontent.com/47536110/86578084-887d0a00-bf7b-11ea-9479-0ca1e26a36f9.png">
on the right.

Reply:
> Why have you removed the onremove call?

Because the map can never initiate marker removal, and all the callbacks are used to transmit actions from the users on a given panel to all others panels. 

I'll look into the bugs you've found!

Reply:
I strongly think we should retain a shadow functionality to remove from the map, it may be something we want to have in the future (happy to discuss in the future)

Reply:
> shadow functionality to remove from the map,

What do you mean by shadow functionality?

Reply:
And if this is something we may want to have in the future, then we can implement it in the future =) But since there is no use case for it now, I would rather keep the code as simple as possible, since that makes future modification easier

Reply:
Regarding the bugs, the `Map.removeMarker` was missing `if (this._is3D()) {this._updateAll3DMarkers();}`, everything should work now.